roles/ceph-osd/defaults/main.yml

```diff
@@ -187,6 +188,18 @@ dedicated_devices: []
 # /dev/sdc1: PARTLABEL="ceph block.wal" PARTUUID="824b84ba-6777-4272-bbbd-bfe2a25cecf3"
 bluestore_wal_devices: "{{ dedicated_devices }}"
 
+# III. Use ceph-volume to create OSDs from logical volumes.
+# Use 'osd_scenario: lvm' to enable this scenario. Currently we only support dedicated journals
+# when using lvm, not collocated journals.
+# lvm_volumes is a dictionary whose key/value pair represent a data lv and a journal pair.
```

Comment:
can we update this comment (and the docs) to mention that any LVM value (like an lv or vg) has to be a *name* and not a path?